scene/3d/reflection_probe.cpp

```diff
@@ -85,38 +85,38 @@ float ReflectionProbe::get_lod_threshold() const {
 	return lod_threshold;
 }
 
-void ReflectionProbe::set_extents(const Vector3 &p_extents) {
-	extents = p_extents;
+void ReflectionProbe::set_size(const Vector3 &p_size) {
+	size = p_size;
 
 	for (int i = 0; i < 3; i++) {
-		if (extents[i] < 0.01) {
-			extents[i] = 0.01;
+		if (size[i] < 0.01) {
+			size[i] = 0.01;
 		}
 
-		if (extents[i] - 0.01 < ABS(origin_offset[i])) {
-			origin_offset[i] = SIGN(origin_offset[i]) * (extents[i] - 0.01);
+		if ((size[i] - 0.01) / 2 < ABS(origin_offset[i])) {
+			origin_offset[i] = SIGN(origin_offset[i]) * (size[i] - 0.01) / 2;
 		}
 	}
 
-	RS::get_singleton()->reflection_probe_set_extents(probe, extents);
+	RS::get_singleton()->reflection_probe_set_size(probe, size);
 	RS::get_singleton()->reflection_probe_set_origin_offset(probe, origin_offset);
 
 	update_gizmos();
 }
 
-Vector3 ReflectionProbe::get_extents() const {
-	return extents;
+Vector3 ReflectionProbe::get_size() const {
+	return size;
 }
 
 void ReflectionProbe::set_origin_offset(const Vector3 &p_extents) {
 	origin_offset = p_extents;
```

Comment:
I couldn't work out what to do with this; last time I just left it. Is `p_extents` referring only to the offset of the origin, or is it somehow related to the old `extents` of the probe itself? Either way, I think it should be replaced as the meaning of 'extents' as 'half of the size' could be a source of confusion.

Reply:
It looks like it is likely a copy-paste error from when the property was first introduced. Renaming the parameter ``p_offset`` would be fine

Reply:
Thanks, that would make more sense; I'll wait until I know how `restore.position` and `.size` work though before force-pushing again, so it's done at once.